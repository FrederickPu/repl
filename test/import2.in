{"cmd": "import Lean\ndef f := 37"}

