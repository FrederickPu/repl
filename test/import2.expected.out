{"results":
 [{"source": "import Lean\n  \n  ", "env": 0},
  {"source": "def f :=\n  37", "env": 1}]}

